internal/provider/nordvpn/updater/models.go

```diff
@@ -83,6 +85,18 @@ func (s *serverData) region() (region string) {
 	return ""
 }
 
+// categories returns the list of categories for the server.
+func (s *serverData) categories() (categories []string) {
+	cats := make([]string, 0, len(s.Groups))
+
```

Comment:
nit just for the sake of using the returned variable and not create another 😉 

```go
	categories = make([]string, 0, len(s.Groups))
```

You would have to also update `cats` -> `categories`

Reply:
I'm not sure what you mean here? Are you saying to rename cats to categories?
![image](https://github.com/qdm12/gluetun/assets/18214951/a0f723b2-7d3f-4b5f-aa3c-bd28176028f4)

If I were to then remove the `make` it would add dupes to categories. Or are you saying to instead remove from categories and invert the IF logic?

Reply:
No just have it as:

```go
func (s *serverData) categories() (categories []string) {
	categories = make([]string, 0, len(s.Groups))
```

and that will work 😉 `:=` is used to declare new variables, but `categories` is already there as the return variable.
If you use `make([]string, 0, len(s.Groups))`, the capacity of the slice is len(s.Groups), but the length is 0, so appending will start at index 0, then 1 etc.